### PR TITLE
Add Sakigw to the list of moderators

### PR DESCRIPTION
**Pull request by Sakisan**

### CODE REVIEW

File: config/environment.rb [hunk 1]
```diff
@@ -17,7 +17,8 @@
   'federicogalassi',
   'timvisher',
   'udioica',
-  'braxler'
+  'braxler',
+  'Sakigw'
```
Comment (igrigorik) [inline config/environment.rb]: I don't believe it should be a problem, but for consistency can we lowercase?

Comment (Sakisan) [reply config/environment.rb]: If it is not a problem then sure, but I don't know about that. The URL http://www.vimgolf.com/Sakigw shows the user page while http://www.vimgolf.com/sakigw does not.

**Comment by igrigorik** [comment]
Sanity check: http://www.vimgolf.com/sakigw — doesn't turn up an account?

**Comment by igrigorik** [comment]
Insert triple facepalm here.

http://www.vimgolf.com/sakigw - doesn't turn up an account.
http://www.vimgolf.com/Sakigw - does.

Sorry about the noise, can you revert back to cap?

**Comment by igrigorik** [comment]
Should be live! Thanks for your patience.

**Pull request merged**